### ISSUE TITLE
while creating an event when we click invite attendees, the window doesn't have scroll bar to see the full calendar area.

### ISSUE THREAD

**Issue by vikas290691** [Ericsson/exchangecalendar#266]
Steps to reprduce:
1. Click on new event creation.
2. Click on invite attendees.
3. One window will open which will show the calendar area for you and where you can see others calendar as well.

issue: there is no ,horizontal scroll bar to check the calendar area. 

Note: The same scroll bar comes when we maximize the same window.

**Comment by kiravi** [comment]
There are arrows on top to scroll left/right  (says next time slot etc.)

**Comment by vikas290691** [comment]
yes , thanks for the info. I knew that but what I mean, if scrolling functionality is there, when we maximize it , that too it (scrolling) should be there also before resize it, logically.
And you may not be able to use previous slot first time. Anyway thats not require functionality. But if any user want to see that also, then it will be difficult to him to see the previous slot.
Please check it and let me know your inputs.

**Comment by bavincen** [comment]
Scrollbar is not hidden here
XUL:STACK flexing has issu
may be some css change needed for freebusy-container

have a look at the code of free-busy container

code;
 <vbox id="freebusy-container" persist="width">
      <stack flex="1">
        <vbox flex="1">
          <freebusy-timebar id="timebar"
                            range="16"/>
          <freebusy-grid flex="1"
                         id="freebusy-grid"
                         range="16"/>
        </vbox>
        <selection-bar id="selection-bar"
                       range="16"/>
      </stack>
      <scrollbar orient="horizontal"
                 id="horizontal-scrollbar"
                 maxpos="100"/>
    </vbox>

**Comment by tfft** [comment]
Scroll-bars are similarly missing from the "Edit Event" window.  If the description is beyond the visible bottom box, no scroll-bar appears though one will be able to scroll down if one presses within the box and uses the cursor to scroll down.  This needs to be fixed as one will not know ahead of time that there is more to the meeting invite's description which is confusing.

**Comment by bavincen** [comment]
@tfft seach for calendar-event-dialog
i guess you will get that window code

**Comment by tfft** [comment]
I'm sorry, search for it where ?  Is there a setting I'm missing somewhere to enable this ?  If so, why is it not enabled by default ?
